### DISCUSSION TITLE
readonly的情况下，如何关闭表格的hover-bar

### DISCUSSION THREAD

**Discussion by ChenKun1997** [wangeditor-team/wangEditor#4447] [Q&A]
![image](https://user-images.githubusercontent.com/24249578/176374173-8f680c7a-48f6-4fa1-a5e1-963ec85e80f4.png)
如图，在readonly模式下，怎么禁止选中表格弹出这个hoverbar

**Comment by wangfupeng1988** [comment]
https://github.com/wangeditor-team/wangEditor/issues/4364 已记录下，等待优化
